app/services/transfers/service.go

```diff
@@ -353,7 +358,9 @@ func (ts *Service) TransferData(txId string) (service.TransferData, error) {
 		signatures = append(signatures, m.Signature)
 	}
 
-	requiredSigCount := len(ts.contractsService.GetMembers())/2 + 1
+	// TODO: remove mockChainID and add TargetChainID to the parameters of t (*entity.Transfer)
+	mockChainID := big.NewInt(1)
+	requiredSigCount := len(ts.contractServices[mockChainID].GetMembers())/2 + 1
```

Comment:
TODO: We have to add new issue in zenhub -> Instead of getting a harcoded value for required signatures, we need to read them from the contract. The contract will have that information. In the case for Hedera it can be in the env config